.gitignore

```
@@ -2,3 +2,6 @@ docker/mysql/data
 phpmyadmin
 .env
 .vscode
+auction/
+*
+!.gitkeep
```

パス指定が無くて乱暴なので、パス指定しましょう
あと*は不要ですね
サジェストをコピペOKです

```suggestion
html/mycakeapp/webroot/img/auction/*
!html/mycakeapp/webroot/img/auction/.gitkeep
```